source/utilities.tex

```diff
@@ -16908,6 +16939,27 @@
  Each type in the template parameter pack \tcode{T} shall be
  complete, \cv{}~\tcode{void}, or an array of unknown bound. \\ \rowsep
 
+\indexlibrary{\idxcode{basic_common_reference}}%
+\tcode{template <class, class,}
+ \hspace*{2ex}\tcode{template <class> class,}
+ \hspace*{2ex}\tcode{template <class> class>}
+ \tcode{struct}
+ \hspace*{2ex}\tcode{basic_common_reference;}
+ &
+ A program may specialize this trait if at least one
+ template parameter in the specialization depends on a user-defined
```

Comment:
LWG 2139 (to be applied in #2193) now uses "program-defined type" here.

Reply:
I'll update uses of "user-defined type" accordingly.